server/controllers/UsersController.js

```diff
@@ -1,9 +1,79 @@
-import models from '../models/Users';
+import { Client } from 'pg';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+import database from '../models/database';
 
-class UsersController {
-  static createUsers(req, res) {
-    models.userSignUp(req, res);
+class Users {
+  static userSignUp(req, res) {
+    const connectString = database;
+    const clientString = new Client(connectString);
+    clientString.connect();
+
+    const hashedPassword = bcrypt.hashSync(req.body.password, 10);
+
+    // Check if user exist
+    const query = clientString.query(`${'insert into users (fullname,email,password,department,role) ' +
```

Comment:
The query string can be a variable, instead of just passing the string to the query function